polylogue/_vendor/frontmatter.py

```
@@ -0,0 +1,65 @@
+        if len(parts) != 2:
+            continue
+        key = parts[0].strip()
+        value = parts[1].strip()
+        if value.startswith("\"") and value.endswith("\""):
+            metadata[key] = value[1:-1]
+        elif value.lower() in {"true", "false"}:
+            metadata[key] = value.lower() == "true"
+        else:
+            try:
+                metadata[key] = json.loads(value)
+            except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)